stdlib/ballerina-builtin/src/main/ballerina/auth.authz.permissionstore/permission-store.bal

```diff
@@ -37,7 +37,7 @@ public function <PermissionStore permissionStore> isAuthorized (string username,
 public function <PermissionStore permissionStore> readGroupsOfScope (string scopeName) returns (string) {
     error e = {message:"Not implemented"};
     throw e;
-    return null;
+    return "";
```

Comment:
Can't reach the return right?

Reply:
Yes, but compilation fails if this is not there.